pkg/controller/v1alpha1/localmodelnode/controller_test.go

```
@@ -52,7 +52,8 @@ func (m *MockFileInfo) Info() (fs.FileInfo, error) { return nil, nil }
 type mockFileSystem struct {
 	FileSystemInterface
 	// represents the dirs under models root
-	subDirs []os.DirEntry
+	subDirs             []os.DirEntry
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)